### PR TITLE
Add sharing UI buttons to profile and canvas.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6619/

**Comment by kadams54** [comment]
## QA

Since this deals with the profile page, it's probably best to QA using guiproxy. If you don't have it:

~~~~
go get github.com/frankban/guiproxy
make run
guiproxy
~~~~

Then visit the GUI running at http://localhost:8042.

Visit the profile and canvas screens. Verify that the share buttons are not present under normal conditions.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6620/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6621/

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 1]
```diff
@@ -39,11 +36,24 @@
...
-    display: none;
+  &--initial {
+    &__export,
+    &__share {
+      display: none;
+    }
   }
 
-  &--initial &__import {
+  .link {
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: Is .link a bit generic? Or is it a standard pattern?

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 2]
```diff
@@ -1,14 +1,15 @@
 // scss-lint:disable IdSelector
-#import-export-container { // sass-lint:disable-line no-ids, id-name-format
+#model-actions-container { // sass-lint:disable-line no-ids, id-name-format
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: Home come ID's?

**Comment by Lukewh** [comment]
LGTM :+1: just two questions for my own understanding

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss (hunk 2 above)
Comment (kadams54) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: To be honest, I'm not sure. It's what was there before; I just updated to match the new name (`s/import-export/model-actions/`).

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss (hunk 1 above)
Comment (kadams54) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: The short answer is yes: it's a standard pattern. That said, I'm not sure where the base style originates, but it applies the following styles to the elements:

~~~~
cursor: pointer;
color: #333;
text-decoration: underline;
~~~~

**Comment by kadams54** [comment]
Thanks @Lukewh and @bac! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**